<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mbs_Widget</class>
 <widget class="QWidget" name="Mbs_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modbus Station</string>
  </property>
  <widget class="QTabWidget" name="mbs_tab_widget_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="serial_info_gb_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>781</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Port Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>751</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="serial_port_lbl_">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serial_port_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyAMA0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttya</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/dev/ttyb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM10</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="baud_lbl_">
       <property name="text">
        <string>Baud</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baud_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="data_bits_lbl_">
       <property name="text">
        <string>DataBits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="data_bits_cb_">
       <item>
        <property name="text">
         <string>8 bits</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 bits</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="parity_lbl_">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="parity_cb_">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stop_bits_lbl_">
       <property name="text">
        <string>Stop Bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stop_bits_cb_">
       <item>
        <property name="text">
         <string>1 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5 bits</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 bits</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="flow_ctrl_lbl_">
       <property name="text">
        <string>Flow Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="flow_ctrl_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hardware</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>software</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="conn_lbl_">
        <property name="text">
         <string>Connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="conn_cb_">
        <item>
         <property name="text">
          <string>Serial Port</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="type_lbl_">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type_cb_">
        <item>
         <property name="text">
          <string>Master</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slave</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode_lbl_">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mode_cb_">
        <item>
         <property name="text">
          <string>RTU</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="start_btn_">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stop_btn_">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="tcp_info_gb_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>781</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>TCP Port Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>751</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="slave_ip_lbl_">
       <property name="text">
        <string>Slave IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="slave_ip_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192.168.100.105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ip_port_lbl_">
       <property name="text">
        <string>IP Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ip_port_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>502</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeout_lbl_">
       <property name="text">
        <string>Timeout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="timeout_cb_">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>5000 ms</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="tcp_info_gb_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>TCP Port Info</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>21</y>
       <width>751</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="slave_ip_lbl_1">
        <property name="text">
         <string>Slave IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="slave_ip_cb_1">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>192.168.100.105</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ip_port_lbl_1">
        <property name="text">
         <string>IP Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ip_port_cb_1">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>502</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeout_lbl_1">
        <property name="text">
         <string>Timeout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="timeout_cb_1">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>5000 ms</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
